test/executor/create_test.cpp

```diff
@@ -39,6 +39,35 @@ namespace test {
 
 class CreateTests : public PelotonTest {};
 
+TEST_F(CreateTests, CreatingDB) {
+    // Bootstrap
+    auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+    auto txn = txn_manager.BeginTransaction();
+  
+    // Create plans with database name set.
+    planner::CreatePlan node("PelotonDB", CreateType::DB);
+
+    std::unique_ptr<executor::ExecutorContext> context(
+        new executor::ExecutorContext(txn));
+    // Create executer
+    executor::CreateExecutor executor(&node, context.get());
+  
+    executor.Init();
+    executor.Execute();
+  
+    EXPECT_EQ(0, catalog::Catalog::GetInstance()
+                     ->GetDatabaseObject("PelotonDB", txn)
+                     ->database_name
+                     .compare("PelotonDB"));
+
+    txn_manager.CommitTransaction(txn);
```

Comment:
This only checks the existence of the DB inside the same transaction. It would be nice to start a new transaction to check again to ensure durability.

Reply:
Agree